src/mudlet-lua/lua/CoreMudlet.lua

```diff
@@ -8,1916 +8,2034 @@
+  ---   <pre>
+  ---   x,y = calcFontSize(10)
+  ---   </pre>
+  --- @usage Create a miniConsole that is 45 letters in length and 25 letters in height for font size 7
+  ---   <pre>
+  ---   font_size = 7
+  ---   local x, y = calcFontSize( font_size )
+  ---   createMiniConsole("map",0,0,0,0)
+  ---   setMiniConsoleFontSize("map", font_size)
+  ---   resizeWindow( "map", x*45, y*25 )
+  ---   </pre>
+  function calcFontSize(fontSize)
+  end
+
+
+
+  --- The <i>channel102 table</i> is used by Aardwolf mud for returning various information about you state. <br/>
+  --- Read <i>http://www.aardwolf.com/blog/2008/07/10/telnet-negotiation-control-mud-client-interaction/</i>
+  --- page for details.
+  ---
+  --- @usage Display content of channel10 table.
+  ---   <pre>
+  ---   display(channel102)
+  ---   </pre>
+  --- @usage Function for detecting AFK status on Aardwolf mud.
+  ---   <pre>
+  ---   function amIafk()
+  ---      for k,v in pairs(channel102) do
+  ---         if k==100 and v==4 then
+  ---            return true
+  ---         end
+  ---      end
+  ---      return false
+  ---   end
+  ---   </pre>
+  ---
+  --- @see sendTelnetChannel102
+  ---
+  --- @class function
+  --- @name channel102
+  channel102 = {}
+
+
+
+  --- Clears the user window or a mini console with the name given as argument.
+  --- <b><u>TODO</u></b> is this still working?
+  function clearUserWindow(windowName)
+  end
+
+
+
+  --- <b><u>TODO</u></b>  clearWindow - TLuaInterpreter::clearUserWindow
+  function clearWindow(windowName)
+  end
+
+
+
+  --- Clears the user window or a mini console with the name given as argument.
+  ---
+  --- @param windowName optinal
+  function closeUserWindow(windowName)
+  end
+
+
+
+  --- The <i>command variable</i> holds initial user command e.g. unchanged by any aliases or triggers.
+  --- This is typically used in alias scripts.
+  ---
+  --- @see line
+  ---
+  --- @class function
+  --- @name command
+  command = ""
+
+
+
+  --- Copies the current selection to the clipboard. This function operates on rich text, i.e.
+  --- the selected text including all its format codes like colors, fonts etc. in the clipboard
+  --- until it gets overwritten by another copy operation. example: This script copies the current
+  --- line on the main screen to a user window (mini console) named chat and gags the output on
+  --- the main screen.
+  ---
+  --- @usage
+  ---   <pre>
+  ---   selectString(line)
+  ---   copy()
+  ---   appendBuffer("chat")
+  ---   replace("This line has been moved to the chat window!")
+  ---   </pre>
+  ---
+  --- @see paste
+  --- @see appendBuffer
+  function copy()
+  end
+
+
+
+  --- Creates a named buffer for formatted text, much like a user terminal window, but the buffer
+  --- cannot be shown on the screen; intended for temporary buffer work
+  function createBuffer(name)
+  end
+
+
+
+  --- <b><u>TODO</u></b>  createButton - TLuaInterpreter::createButton
+  function createButton()
```

Comment:
This (brokenly IMHO) is an effective alias for `createLabel()` which is a pity because it means we cannot use it to, um, create Buttons (to go on Toolbars or Menus)... 😞